website/content/docs/upgrading/upgrade-to-1.12.x.mdx

```diff
@@ -16,3 +16,59 @@ for Vault 1.12.x compared to 1.11. Please read it carefully.
 Vault Enterprise will now perform a supported storage check at startup. There is no impact on open-source Vault users.
 
 @include 'ent-supported-storage.mdx'
+
+## Known Issues
+
+### Pinning to builtin plugin versions may cause failure on upgrade
+
+1.12.0 introduced plugin versions, and with it, the ability to explicitly specify
+the builtin version of a plugin when mounting an auth, database or secrets plugin.
+For example, `vault auth enable -plugin-version=v1.12.0+builtin.vault approle`. If
+there are any mounts where the _builtin_ version was explicitly specified in this way,
+Vault may fail to start on upgrading to 1.12.1 due to the specified version no
+longer being available.
+
+To check whether a mount path is affected, read the tune information, or the
+database config. The affected plugins are `snowflake-database-plugin@v0.6.0+builtin`
+and any plugins with `+builtin.vault` metadata in their version. In this example,
```

Comment:
> The affected plugins are `snowflake-database-plugin@v0.6.0+builtin`
and any plugins with `+builtin.vault` metadata in their version.

When I first read this I thought it was saying the only plugins affected by this known issue were the snowflake plugin and the builtin.vault plugins, but I think this is just referring to the example shell-session below? Maybe we could clarify that this is referring to the example if that's the case? If not, this sentence may fit better in the previous paragraph.

Reply:
Your first interpretation was correct. e.g. kubernetes auth isn't affected because its builtin version hasn't changed between 1.12.0 and 1.12.1. I can add that as a third example to help clarify?

Reply:
Ah ok, and maybe mention the scope in the first paragraph since that seems to indicate that all plugins are affected.

Reply:
Yeah, sorry, by "all plugins" I meant all builtin plugins, and I think my confusion comes from the line 

> If there are any mounts where the _builtin_ version was explicitly specified in this way,
Vault may fail to start ...

which is later qualified in the subsequent paragraph to be only `builtin.vault` or snowflake.

I think what you have in #17799 clears it up 👍